addon/components/nav-modal.js

```diff
@@ -1,36 +1,48 @@
 import Ember from 'ember'
 import layout from '../templates/components/nav-modal'
-import _ from 'lodash'
 
-export default Ember.Component.extend({
-  nav: Ember.inject.service('frost-navigation'),
+const {
+  Component,
+  computed,
+  observer,
+  inject,
+  A
+} = Ember
+
+export default Component.extend({
+  frostNavigation: inject.service(),
 
   classNames: ['nav-modal'],
   layout,
   tabindex: 0,
   attributeBindings: ['tabindex'],
   activeCategory: null,
-  _categoryChanged: Ember.observer('nav._activeCategory', function () {
-    this.set('activeCategory', this.get('nav._activeCategory'))
+  _categoryChanged: observer('frostNavigation._activeCategory', function () {
+    this.set('activeCategory', this.get('frostNavigation._activeCategory'))
```

Comment:
Is this consumed in a downstream template?  If so, you may want to wrap it in a `scheduleOnce('sync',`